### ISSUE TITLE
Advanced Filter Dialog generates wrong syntax for Name filter

### ISSUE THREAD

**Issue by Zenoguy3** [dekkerglen/CubeCobra#407]
On advanced filter dialog, Card name field should generate "name: ..." for the name filter, but instead generates "n: ..." and therefore displays an invalid search string error.
The "name: ..." syntax works properly.

**Comment by hallc** [comment]
Please assign this to me.

**Comment by hallc** [comment]
I can no longer reproduce this issue on `master`.  The advanced filter dialog is generating the correct syntax from both the Edit Cube and Compare screens.  I suspect that this issue was fixed in #470.

However, the problematic bit from `editcube.js` is still present:

```javascript
$('#applyAdvancedFilterButton').click(function(e) {
  e.preventDefault();

  var str = '';

  if ($('#filterName').val().length > 0) {
    str += 'n:' + $('#filterName').val();
  }
```

It looks like that method is no longer called.  Did the react conversion leave us with some dead filtering code that can be removed?

/cc @phulin

**Comment by phulin** [comment]
Yep, that code is dead and can be removed. I'll submit a cleanup PR after
the release.
